lms/static/js/instructor_dashboard/student_admin.js

```diff
@@ -439,6 +433,12 @@
                     })
                 });
             });
+            var $btn_download_blank_lti = this.$section.find("input[name='lti-blank-grades']");
```

Comment:
This can be a local variable and doesn't need to be assigned to `this`.

Reply:
Cool